src/plugins/tool_extrude.py

```diff
@@ -29,128 +29,102 @@ def __init__(self, api):
             "Extrude", None)
         self.action.setStatusTip("Extude region")
         self.action.setCheckable(True)
-        self.action.setShortcut(QtGui.QKeySequence("Ctrl+9"))
+        self.action.setShortcut(QtGui.QKeySequence("Ctrl+0"))
```

Comment:
Does this shortcut work for you?
At least it's not working on OS X (it does nothing and I see this warning in the console: `QAction::eventFilter: Ambiguous shortcut overload: ?0`)